web/index.html

```diff
@@ -1192,7 +1192,7 @@ <h4 class="modal-title" id="switchRegistryModalLabel">Switch Netdata Registry Id
 
 	'disk': {
 		title: 'Disks',
-		info: undefined
+		info: 'Charts with performance information for all the system disks. Special care has been given to present disk performance metrics in a way compatible with <code>iostat -x.</code> netdata by default prevents rendering performance charts for individual partitions. Partitions can still be enabled by altering the relative settings in the netdata configuration file.'
```

Comment:
(also move the dot outside the `code` block - it seems like it is part of the `iostat -x` command.